### PR TITLE
add the old idv1symbol to be v2.

### PR DESCRIPTION
**Pull request by Matyeyev**
the old version of idv1 symbol keyboard layout by @awandepan in 1.4 version before it was changed in 1.7.

### CODE REVIEW

**Comment by dessalines** [comment]
Check the keyboard addition PRs for examples. You also need to add this new key to types.kt and a section to utils.kt.

**Comment by awandepan** [comment]
@Matyeyev the old version i created before i read thoroughly the keyboard layout guideline, now i type smoothly with the new version (fix some letter position align with keyboard layout guide in readme),

**Review by Matyeyev — COMMENTED** [review]
is these commit qualified? let me know in i need to edit anything.

**Comment by Matyeyev** [comment]
> the old version of idv1 symbol keyboard layout by @awandepan in 1.4 version before it was changed in 1.7.

can i make the old layout to be v2 instead of legacy v1. because i think the name will get conflicted (no keyboard layout with that name).

**Comment by dessalines** [comment]
Just rename one of them v1, and the other v2.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyIDv1LegacySymbols.kt
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyIDv1LegacySymbols.kt:1]: Rename this file to v1 or v2, not `Legacy`

**Review by dessalines — COMMENTED** [review]
Also run `./gradlew formatKotlin` to pass lint.

**Review by dessalines — APPROVED** [review]
LGTM, thx!

**Comment by dessalines** [comment]
Failing lint again.

**Comment by Matyeyev** [comment]
> Failing lint again.

what does this mean? i don't understand about this. I only edit those three files. is there something wrong with my code?

**Comment by dessalines** [comment]
run `./gradlew formatKotlin`

**Pull request merged**